src/app.ts

```diff
@@ -10,6 +10,7 @@ import { bundlify } from './helpers/bundler';
 program
   .requiredOption('-i --ig <path-to-ig>', 'Path to full IG directory')
   .option('-o --output <path-to-output-dir>', 'Path to desired output directory', './output')
+  .option('-o --libname <path-to-library>', 'Path to library', './libname')
```

Comment:
```suggestion
  .option('-o --libname <path-to-library>', 'Path to library')
```

we don't want to have a default for this. If it's omitted, it will just be `undefined` and will be set to the IG json accordingly.

If you run this without the libname argument, it doesn't behave properly. Removing this default should get us there.